modules/core/src/test/scala/gem/math/OffsetPSpec.scala

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import gem.arb._
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{ FlatSpec, Matchers }
+
+import scalaz.Monoid
+
+class OffsetPSpec extends FlatSpec with Matchers with PropertyChecks {
+  import ArbOffset._
+
+  // Compilation test
+  protected val a0 = implicitly[Monoid[Offset.P]]
+
+  "Conversion to angle" must "be invertable" in {
+    forAll { (p: Offset.P) =>
+      Offset.P(p.toAngle) shouldEqual p
+    }
+  }
+
+  "Offset.P forms an Abelian Group over addition. It" must "be associative" in {
```

Comment:
Unrelated to the PR, but github doesn't know how to color code this type of code, confusing...